### PR TITLE
Update Test Cases

### PR DESCRIPTION
**Pull request by maj0-0**
remove tag from everywhere
add typical tasks
jsonadaptedtask and taskexception tests

### CODE REVIEW

**Review by wnchan — COMMENTED** [review]
LGTM!

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,18 @@ checkstyle {
     toolVersion = '10.2'
 }
 
+run {
+    enableAssertions = true
+}
+
 test {
+    doFirst {
```
Comment (neyapraveen) [inline build.gradle:28]: Thanks for handling the ubuntu problem for fx tests!

File: build.gradle [hunk 2]
```diff
@@ -20,7 +20,18 @@ checkstyle {
...
+        if (System.getProperty("os.name").equals("Linux")) {
+            exclude '**/ui/**'
+            // Excluded because it fails on Ubuntu
```
Comment (neyapraveen) [inline build.gradle:31]: Maybe can say FX tests do not successfully run on Ubuntu Runner coz clearCommand uses fx too

**Review by neyapraveen — COMMENTED** [review]
LGTM!!

**Review by Bearypop — COMMENTED** [review]
Nice!

File: src/test/java/seedu/address/model/socialmedialink/SocialMediaLinkTest.java [hunk 3]
```diff
@@ -16,7 +14,7 @@ public void constructor_null_throwsNullPointerException() {
     @Test
     public void constructor_invalidSocialMediaLink_throwsIllegalArgumentException() {
         String invalidSocialMediaLink = "";
-        assertThrows(IllegalArgumentException.class, () -> new Tag(invalidSocialMediaLink));
+        assertThrows(IllegalArgumentException.class, () -> new SocialMediaLink(invalidSocialMediaLink));
```
Comment (PearlynnT) [inline src/test/java/seedu/address/model/socialmedialink/SocialMediaLinkTest.java:17]: Thanks for correcting the mistake!

**Pull request merged**